### ISSUE TITLE
Capture pcap 

### ISSUE THREAD

**Issue by LPardue** [ekr/qdrive#1]
In the case of some automated testing via CI, it might be helpful if the test could store a pcap in a manner that can be checked after the fact. This might me exporting and storing the key to decrypt the pcap later, or storing a decrypted pcap.

**Comment by alagoutte** [comment]
Yes, good idea to save the pcap :)

About storing the key, need to may be add new type on SSLKEYLOGFILE.. @Lekensteyn

**Comment by Lekensteyn** [comment]
I assume this is about IETF QUIC and not the Google variant (given that it's in ekr repo)?

**Comment by LPardue** [comment]
Correct, with TLS 1.3 powered handshake

**Comment by ekr** [comment]
Yes, though there's nothing in *this* code that wouldn't let you test gQUIC either.... (at least yet)
